EEE
======================================================================
ERROR: test_collect_and_send_data_with_empty_response (__main__.TestModbusClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "d:\OneDrive - LTTS\Desktop\ModbusTcp\test.py", line 61, in test_collect_and_send_data_with_empty_response
    collect_and_send_data(client, send_to_iothub_mock)
TypeError: collect_and_send_data() takes 1 positional argument but 2 were given

======================================================================
ERROR: test_collect_and_send_data_with_invalid_data (__main__.TestModbusClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "d:\OneDrive - LTTS\Desktop\ModbusTcp\test.py", line 44, in test_collect_and_send_data_with_invalid_data
    collect_and_send_data(client, send_to_iothub_mock)
TypeError: collect_and_send_data() takes 1 positional argument but 2 were given

======================================================================
ERROR: test_collect_and_send_data_with_valid_data (__main__.TestModbusClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "d:\OneDrive - LTTS\Desktop\ModbusTcp\test.py", line 27, in test_collect_and_send_data_with_valid_data
    collect_and_send_data(client, send_to_iothub_mock)
TypeError: collect_and_send_data() takes 1 positional argument but 2 were given

----------------------------------------------------------------------
Ran 3 tests in 0.001s

FAILED (errors=3)
